repos/rust-lang/wg-allocators.toml

```diff
@@ -0,0 +1,7 @@
+org = "rust-lang"
+name = "wg-allocators"
+description = "Home of the Allocators working group: Paving a path for a standard set of allocator traits to be used in collections!"
+bots = []
+
+[access.teams]
+wg-allocators = "write"
```

Comment:
Nit: this might be better as admin since this is just meant as a place for the wg to convene, but I'm fine with trying it out with the lower permission.

Reply:
Good point. I think that `maintain` should be enough though, to edit wikis, manage projects etc.? Without destructive actions.